src/scanpy/preprocessing/_highly_variable_genes.py

```diff
@@ -538,6 +573,7 @@ def highly_variable_genes(  # noqa: PLR0913
     subset: bool = False,
     inplace: bool = True,
     batch_key: str | None = None,
+    _filter_genes: bool = False,
```

Comment:
this parameter is unused, what’s the plan here?

if this isn’t accidentally left in: if you want to introduce a new public parameter, it can’t have a name prefixed with `_`, if it’s for internal use only, it shouldn’t have documentation

Reply:
I’m moving it into a new PR so this one can go into a patch